Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/2_Metagaming.xml

```diff
@@ -1,14 +1,131 @@
-  - Do not use in-character channels to bypass a lack of ability to use OOC/LOOC chats (e.g. "Huh, wonder where the captain went? OOC He probably left to get dinner lol").
-  - Do not engage in meta-communications (i.e. using external channels to communicate with other players in the same game).
-    - Do not stream, discuss, or share information about the current round to the Delta-V Discord.
-  - You are allowed to have knowledge of past experiences and prior shifts, and players are allowed to have in-character relationships (friends, enemies, or otherwise). However, this cannot be used as a sole reason to grant or deny items/accesses/favors based exclusively on having a relationship with one another (i.e. meta-friending/meta-grudging).
-    - Antagonists in previous rounds must not be treated differently due to their prior antagonist status. Notwithstanding, confession of crimes committed in a prior shift are still admissible in court.
-    - Additionally, and for the sake of continuity, your character will never have permanently died in previous rounds.
-  - Do not "antag roll." This is the act of joining rounds for the purpose of seeing if you have received an antagonist role, and leaving soon after if not.
+  # Rule B2: Follow the Metashield.
+  - Something that is "shielded" cannot be known by your character during a round until the "revealing condition" happens. This also means that your character cannot do things based on "shielded" information. Knowing or acting on something that is shielded before the revealing condition is met is referred to as metagaming.
+
+  - Revealing conditions reveal the shielded information for the round, not for a specific instance. This means that once a revealing condition is met in a round, the shield no longer applies in any case for the remainder of the round.
+
+  ## Never Revealed IC
+  Some shields are never revealed IC. This means that your character can never act as if they know about that shielded thing.
+
+  The following are shielded:
+  - Current game mode and possible antags during the current game mode.
+  - Events from previous rounds.
+  - Events you experienced as a different character.
+  - All information related to the player of a character rather than the character itself. (See "Metafriending and Metagrudging" below.)
+  - All information gained while dead or a ghost.
+  - Anything related to Code Epsilon or Deathsquads
+
+  This does not prevent knowing that a shift will end, but does prohibit things like preparing to kill people at central command when roleplay rules stop being enforced on LRP.
+
+ ## Previous Rounds
+
+  Events from previous rounds are not shielded, but they should not hold any bearing on the current round. (i.e. you cannot be arrested for crimes in a past round)
```

Comment:
it says a few lined above that events from previous rounds are always shielded

Reply:
Got it 👍